app/src/main/java/com/duckduckgo/app/global/DuckDuckGoActivity.kt

```diff
@@ -56,7 +56,7 @@ abstract class DuckDuckGoActivity : AppCompatActivity() {
 
     override fun onDestroy() {
         themeChangeReceiver?.let {
-            LocalBroadcastManager.getInstance(applicationContext).unregisterReceiver(it)
+            androidx.localbroadcastmanager.content.LocalBroadcastManager.getInstance(applicationContext).unregisterReceiver(it)
```

Comment:
I don't see a package conflict. Can we remove `androidx.localbroadcastmanager.content`?